src/main/java/uk/gov/hmcts/payment/processors/DeadLetterQueueProcessor.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.payment.processors;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import java.util.HashMap;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DeadLetterQueueProcessor implements JobProcessor {
+    private static final Logger LOG = LoggerFactory.getLogger(DeadLetterQueueProcessor.class.getName());
+    private final Map<String, String> headers = new HashMap<>();
+    @Override
+    public void process(String serviceToken, String baseURL) {
+
+        LOG.info("Value in DeadLetterQueueProcessor-----"+"BaseURL--------"+baseURL);
+        headers.put("ServiceAuthorization", serviceToken);
+        RestAssured.given().relaxedHTTPSValidation()
```

Comment:
Is Rest Assured a standard Dev Library as as far as I know this is to be used in Tests and Feign is used in Application Code...

Reply:
Can you please point me to the standards which prohibit the use of Rest Assured?

Reply:
Not using Test libraries in production deployment or even code is an industry practice at the basic level of action. This need not be strictly defined as an organizational standard.

Reply:
Resolved this point as we have a backlog Card to fix this issue in PAY-5606